tools/report-converter/codechecker_report_converter/report/__init__.py

```diff
@@ -307,6 +320,9 @@ def __init__(
         self.type = type
         self.annotations = annotations
 
+        self.static_message = \
```

Comment:
Do we need a new field? Can we not just change `self.message`?

Reply:
No, because we still want to use the original value of `self.message` to display it for the user. The value of this field goes to the GUI and command line output.